shared/chat/inbox/container/index.js

```diff
@@ -137,10 +140,12 @@ class InboxWrapper extends React.PureComponent<Props, State> {
   }
 
   componentDidUpdate(prevProps) {
-    const loadedForTheFirstTime = prevProps.rows.length === 0 && this.props.rows.length > 0
+    const loadedForTheFirstTime =
+      !prevProps.filter && prevProps.rows.length === 0 && this.props.rows.length > 0
     // See if the first 6 are small, this implies it's expanded
     const smallRowsPlusOne = prevProps.rows.slice(0, 6).filter(r => r.type === 'small')
-    const expandedForTheFirstTime = smallRowsPlusOne.length === 5 && this.props.rows.length > 5
+    // !prevProps.filter because that's a false positive
+    const expandedForTheFirstTime = !prevProps.filter && smallRowsPlusOne.length > 5
```

Comment:
`smallRowsPlusOne === 5` to `smallRowsPlusOne > 5` here, I think this is right because if `smallrowsPlusOne.length <= 5`, we've definitely shown those rows before (before expanding or we have a fresh meta because of incoming activity)